### PR TITLE
update api call with named argument

### PR DESCRIPTION
**Pull request by IanBoyanZhang**
It seems softmax_cross_entropy_with_logits api call is updated in TF1.0

Current code will cause valueError: Only call softmax_cross_entropy_with_logits with named arguments (labels=..., logits=..., ...)

References:
[TensorFlow wavenet issues](https://github.com/ibab/tensorflow-wavenet/issues/223)
[Stackoverflow](http://stackoverflow.com/questions/42296782/valueerror-when-executing-softmax-cross-entropy-with-logits)

### CODE REVIEW

**Comment by tempflip** [comment]
+1; please merge it!

**Comment by mvirgo** [comment]
This got fixed with `f24919e068691502b79e27eb47e87e8c8376c788`, thank you!
